### PR TITLE
iofog cli and service fixes

### PR DESCRIPTION
**Pull request by MaksimChepelev**

### CODE REVIEW

File: src/cli/iofog.js [hunk 1]
```diff
@@ -59,23 +63,22 @@ class IOFog extends BaseCLIHandler {
...
+      { name: 'abs-hw-enable', alias: 'A', type: Boolean, description: 'Enable hardware abstraction on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
+      { name: 'abs-hw-disable', alias: 'a', type: Boolean, description: 'Disable hardware abstraction on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
       { name: 'reboot', alias: 'o', type: Boolean, description: 'Reboot ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
       { name: 'fog-type', alias: 'y', type: Number, description: 'ioFog node architecture type', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'enable', alias: 'e', type: Boolean, description: 'Enable tunnel', group: [constants.CMD_TUNNEL] },
-      { name: 'disable', alias: 'S', type: Boolean, description: 'Disable tunnel', group: [constants.CMD_TUNNEL] },
-      { name: 'info', alias: 'O', type: Boolean, description: 'Display tunnel info', group: [constants.CMD_TUNNEL] },
+      { name: 'tunnel-enable', alias: 'e', type: Boolean, description: 'Enable tunnel', group: [constants.CMD_TUNNEL] },
```
Comment (baghbidi) [inline src/cli/iofog.js]: why did you add `tunnel-` to these? I think `enable`, `disable` and `info` are better.

The command will be:
`fog-controller iofog tunnel --enable`

File: src/sequelize/managers/user-manager.js [hunk 2]
```diff
@@ -59,6 +59,11 @@ class UserManager extends BaseManager {
     });
   }
 
+  // no transaction required here, used by cli decorator
+  findById(id) {
+    return User.find({id: id});
```
Comment (epankou) [inline src/sequelize/managers/user-manager.js]: Looks like here you should pass object like {where: {id:id}}
